tests/network-tests/src/flows/proposals/vetoProposal.ts

```diff
@@ -20,7 +21,7 @@ export default async function vetoProposal({ api, query, lock }: FlowProps): Pro
   const createProposalFixture = new CreateProposalsFixture(api, query, [
     {
       type: 'Signal',
-      details: 'Proposal to be vetoed',
+      details: ('Proposal to be vetoed' as unknown) as Bytes,
```

Comment:
Avoid type assertions

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)